docker-compose.minimal.yml

```diff
@@ -330,6 +330,7 @@ services:
       - OTEL_RESOURCE_ATTRIBUTES
       - OTEL_SERVICE_NAME=loadgenerator
       - PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python
+      - FLAGD_HOST
```

Comment:
Could you also add `FLAGD_PORT`?

Eventually those env vars will be added to `flagd` and the manual configuration we are currently doing won't be necessary anymore.

Reply:
We also need this on the main `docker-compose.yml`